### PR TITLE
Case insensitive tag storage/searching

### PR DESCRIPTION
**Pull request by cadecairos**
#187

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,7 +62,6 @@
     "sinon": "^1.14.1"
   },
   "engines": {
-    "npm": "^2.5.1",
-    "node": "^0.12.1"
+    "node": "4.2.3"
```
Comment (cadecairos) [inline package.json]: I don't remember making this change... but we needed to ditch 0.12.1 anyways.

**Comment by alicoding** [comment]
R+

File: package.json (hunk 1 above)
Comment (cadecairos) [inline package.json]: it appears I patched master to use 4.2.3, and forgot to push the commit to develop

**Comment by cadecairos** [comment]
@alicoding ty!

**Pull request merged**